lib/web/ui/integration.go

```diff
@@ -639,3 +639,9 @@ type AWSOIDCPingRequest struct {
 	// AWS OIDC integration.
 	RoleARN string `json:"roleArn,omitempty"`
 }
+
+// AWSOIDCDeployEC2ICERequest contains request fields for creating an app server.
+type AWSOIDCCreateAWSAppAccessRequest struct {
+	// Labels added to the app server resource that will be created.
+	Labels map[string]string `json:"labels"`
```

Comment:
Is this the expected format?
I think UI labels are usually formatted as:
```json
{
  "labels":[
    {"name":"my-label1", "value":"my-key1"},
    {"name":"my-label2", "value":"my-key2"},
  ]
}
```

Reply:
i was torn between what the right way to do this. i can format the field from frontend or backend, and i ultimately decided the frontend will format it like done [here](https://github.com/gravitational/teleport/blob/ec3e23dd4114d754a7045773e64b554fa092c38d/web/packages/teleport/src/Discover/AwsMangementConsole/CreateAppAccess/CreateAppAccess.tsx#L51)

it's because the `NewAppServerForAWSOIDCIntegration` expects a map of labels

is that okay? i thought it was more accurate b/c it tells the client that the backend doesnt accept duplicate keys

Reply:
I recall seeing multiple endpoint handlers using `libui.Labels` in both responses and requests.
I'm fine with either approach 👍